webnn/resources/test_data/average_pool2d.json

```diff
@@ -0,0 +1,1368 @@
+{
+  "tests": [
+    {
+      "name": "averagePool2d float32 4D tensor default options",
+      "inputs": {
+        "input": {
+          "shape": [1, 2, 5, 5],
+          "data": [
+            83.46227700254062,
+            33.821312758080836,
+            -76.80760955483348,
+            -29.45969284637431,
+            -46.52967419678973,
+            -45.55183988629725,
+            -31.01262212241673,
+            -64.07866030681286,
+            -9.310889091915442,
+            -41.08586138776409,
+            -73.40547764047231,
+            13.158153901575972,
+            -12.270027157926222,
+            76.39896882134298,
+            93.52596758105801,
+            97.13613783584108,
+            -1.7500290525074718,
+            -61.565025090745664,
+            -54.58282317750749,
+            -48.486631084255414,
+            60.32106196347354,
+            4.079223599205321,
+            32.319089892863815,
+            -41.34668340359866,
+            -6.917796525327361,
+            -49.26129609640228,
+            68.73554887005739,
+            2.418686828435483,
+            80.52496201764376,
+            -55.300870794659176,
+            -59.206964627797646,
+            -95.96083088125917,
+            -18.597523013363613,
+            47.98993621629046,
+            -73.6978064325847,
+            -84.76842495411003,
+            -75.75077583450742,
+            -96.09162719339479,
+            -25.728314640543104,
+            43.86839589244039,
+            13.15480395039694,
+            -48.46195196452694,
+            -52.95915779740401,
+            -14.006732047772985,
+            21.69510814172409,
+            76.66673883399159,
+            48.12142294811815,
+            64.0178574522754,
+            23.612354911278203,
+            -96.46550907705537
```

Comment:
Same comment as with reduceSum about subtraction of nearly equal numbers with mixed input data.